packages/components/src/components/form/Input/Input.tsx

```diff
@@ -198,7 +198,11 @@ const Input = ({
             </InputWrapper>
 
             {bottomText && (
-                <BottomText inputState={inputState} isDisabled={isDisabled}>
+                <BottomText
+                    withIcon={inputState && ['error', 'warning'].includes(inputState)}
```

Comment:
cc @jvaclavik this looks fishy WDYT

Reply:
anyway `withIcon` can be solved inside the `BottomText` same way as the `icon` itself is. 

You are already passing whole `inputState` so no need to solve it on this level.

Reply:
Yes, it looks fishy to me. Not sure if we want to do it for all inputs.